### PR TITLE
Lesson 02 Assignment Submissions

### PR DESCRIPTION
**Pull request by dmartin7856**
Hello, 

This PR is for the three Lesson 02 Assignments.  I apologize for the delay in submitting this - I had to resolve a few errors when committing files to git.  

V/R,

Dominic Martin

### CODE REVIEW

File: lesson02/series.py [hunk 1]
```diff
@@ -1,73 +1,100 @@
...
+	# series = [a, b]
+	for i in range(n-2):
+		c = a + b
+		series.append(c)
+		a = b
+		b = c
+
+	print("The value of term", n, "in the Lucas Series is:", series[n-1])
+		
+def sumIt(n, a, b):
```
Comment (natasha-aleksandrova) [inline lesson02/series.py]: pep8 style guidelines recommend snake_case for var names -> `sum_it`

File: lesson02/series.py [hunk 2]
```diff
@@ -1,73 +1,100 @@
...
+''' This function defines the computation for the summation of all terms
+	in a series, given the first two terms. '''
+	series = [a, b]
+	if n == 1:
+		print("The value of the first term is:", a)
+	if n == 2:
+		print("The sum of the first two terms is:", sum(series))
+	for i in range(n-2):
+		c = a + b 
+		series.append(c)
```
Comment (natasha-aleksandrova) [inline lesson02/series.py]: the indentation is off throughout - make sure to use exactly 4 spaces (no tabs)

**Comment by natasha-aleksandrova** [comment]
@dmartin7856 I saw your question on the discussion board - did you get your git issue resolved?

**Comment by dmartin7856** [comment]
I did.  Thanks!

I think what happened was I added files without committing them and then added them later and tried to commit all.  

My guess is git didn't like that too much.  I ended up removing the added files from my local repo, resetting global and then repeating the workflow of adding and committing files individually and it worked fine.  

I'll be a git expert by the end of the cert program...

**Comment by natasha-aleksandrova** [comment]
Glad you got it worked out! Being a git expert is a great thing 😉

**Pull request merged**